### DISCUSSION TITLE
is it possible to use it with mongodb?

### DISCUSSION THREAD

**Discussion by mshamaseen** [paratestphp/paratest#811] [Q&A] [answered]
I have a BIG project with a lot of tests, I tried to use paratest for it, but the issue is that we are using a mix of MongoDB and Postgress, and it looks like paratest is just creating parallel tests database for Postgress only. is there a way to tell it to do that for multiple drivers?

**Comment by alexander-schranz** [comment, marked as answer]
I think your postgresql database configuration already keeps the `TEST_TOKEN` in mind. But you need todo the same for `MongoDB`. It is based on your framework how you can configure your mongodb connection but you can just post or prefix it with the `TEST_TOKEN` normally:

```php
'mongodb' => [
    'database' => env('DB_DATABASE', 'homestead') . getenv('TEST_TOKEN),
],
```

See official docs: https://github.com/paratestphp/paratest#test-token
